pkg/controller/prpqr_reconciler/prpqr_reconciler.go

```diff
@@ -438,3 +529,69 @@ func generateProwjob(rc injectingResolverClient, defaulter periodicDefaulter, ba
 func metadataFromPullRequestUnderTest(pr v1.PullRequestUnderTest) *api.Metadata {
 	return &api.Metadata{Org: pr.Org, Repo: pr.Repo, Branch: pr.BaseRef}
 }
+
+func createJobRunAggregatorJob(uid, mimickedJob, prpqrName, prpqrNamespace string, defaulter periodicDefaulter, startTime time.Time) (*prowv1.ProwJob, error) {
+	labels := map[string]string{"release.openshift.io/aggregation-id": uid, v1.PullRequestPayloadQualificationRunLabel: prpqrName}
+	spec := &corev1.PodSpec{
+		Volumes: []corev1.Volume{
+			{
+				Name: "pull-secret",
+				VolumeSource: corev1.VolumeSource{
+					Secret: &corev1.SecretVolumeSource{SecretName: "registry-pull-credentials"},
+				},
+			},
+			{
+				Name: "gcs-credentials",
+				VolumeSource: corev1.VolumeSource{
+					Secret: &corev1.SecretVolumeSource{SecretName: "gce-sa-credentials-gcs-publisher"},
```

Comment:
I guess we will need to have another secret just for this case. 
@deads2k Can you let us know the required permissions for the service account?